otx/algorithms/detection/utils/utils.py

```diff
@@ -104,20 +104,36 @@ def generate_label_schema(label_names: Sequence[str], label_domain: Domain = Dom
     return label_schema
 
 
-def get_det_model_api_configuration(label_schema: LabelSchemaEntity, task_type: TaskType, confidence_threshold: float):
+def get_det_model_api_configuration(
+    label_schema: LabelSchemaEntity, task_type: TaskType, confidence_threshold: float, tiling_parameters: Any
+):
     """Get ModelAPI config."""
     omz_config = {}
+    all_labels = ""
     if task_type == TaskType.DETECTION:
         omz_config[("model_info", "model_type")] = "ssd"
```

Comment:
Detection model also can be used for tiling.

Reply:
Yes, but in that case we don't need any extra settings. You could check that by inspecting wrappers and configs.